exporter/otlpexporter/README.md

```diff
@@ -1,14 +1,15 @@
 # OTLP gRPC Exporter
 
+| Status                   |                       |
+| ------------------------ | --------------------- |
+| Stability                | [beta]                |
+| Supported pipeline types | traces, metrics, logs |
+| Distributions            | [core], [contrib]     |
```

Comment:
Does this contradict with https://github.com/open-telemetry/opentelemetry-collector#status?

Reply:
🤔 it's a good question. The status table should include a stability level per signal as per https://github.com/open-telemetry/opentelemetry-collector-contrib#stability-levels
>As such, each component should list its current stability level for each telemetry signal

I'll update the table to reflect this.

Reply:
@bogdandrutu added, PTAL